scripts/tests.sh

```diff
@@ -31,6 +31,8 @@ if [[ -d "tests/" ]]; then
 
   if [[ $test_dir = "jinahub/encoders/text/LaserEncoder" ]]; then
     python -m laserembeddings download-models
+  elif [[ $test_dir = "jinahub/segmenters/VADSpeechSegmenter" ]]; then
+    sudo apt-get update && sudo apt-get install libsndfile1 ffmpeg
```

Comment:
is `ffmpeg` also required in the Dockerfile?

Reply:
I think the docker flow still works without `ffmpeg`. It's probably included in the base image?